data/hai/hai reveal 4 middle.txt

```diff
@@ -1070,46 +1091,189 @@ mission "Hai Reveal [A09] History of Appeasement"
+			`	"Respected Elders," Teeneep begins, "I have extensive evidence of a sustained kidnapping campaign in Hai space, facilitated by pirates you've been arm-"`
+			`	"Enough," says an Elder, raising his hand. "We know about your views, the 'five thousand missing Hai,' and your organization's media connections. You're already risking larger issues with your current actions. We're here to speak to your witness. Captain <last>," he says, turning to you, "tell us your views on these matters."`
+			`	You think carefully of Katya's warning as you prepare to respond.`
+			label "elder questions"
+			choice
+				`	"Why have you been making deals with pirates?"`
+					goto "elders why"
+					to display
+						not "Hai Reveal [A09] History of Appeasement: label elders why"
+				`	"Your secrecy is rapidly degrading, and pirate activity is on the rise despite these deals. Are you sure they're worth sacrificing your morals?"`
+					goto "elders secret"
+				`	"A trusted source has warned me of a hostile diplomatic incident that may happen if the Republic finds out you're arming pirates."`
+					goto "Katya warning"
+			label "elders why"
+			action
+				set "Hai Reveal [A09] History of Appeasement: label elders why"
+			`	The left Elder says, "Secrecy, mostly, to prevent a refugee crisis if the Hai become common knowledge. The bulk of the resources go to a few large pirate groups that control the anarchist worlds. We fund the development and protection of their planets, in return for them ensuring the wrong people don't find us."`
+			`	Another Elder adds, "Most of the deals are much smaller, though: when a pirate gang causes mischief, we pay them to stop. Hostages, theft, raids, and the like. Usually, the same gang never bothers us again." Teeneep perks up at the word "hostages," but doesn't speak.`
+				goto "elder questions"
+			label "elders secret"
+			`	The middle Elder counters, "We cannot be certain that the pirate activity will continue. With the proper trades, we may be able to return to the status quo."`
+			`	"That is a best-case scenario," says the right Elder, "but if matters continue to worsen, we'll change our approach." The other two nod in agreement.`
+				goto diplomats
+			label "Katya warning"
+			`	The Elder on the right disagrees. "At this point, the Republic is too weakened to be a threat. I'm confident they will see reason, for now."`
+			`	Fortunately, the middle Elder contradicts him. "We don't want to risk larger problems with humanity. If the situation continues to worsen, then we'll change our approach." The other two find that argument convincing and nod.`
+			label diplomats
+			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of the 'kidnapping' conspiracy theory, so that ends this meeting." Teeneep is visibly furious, but remains silent as the three exit.`
```

Comment:
```suggestion
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable; she is tightly clutching the table, and appears to be ready to throw it, but she remains silent as the three exit.`
```

Reply:
This is waaaay more extreme of a reaction than I was going for. She's supposed to be largely under control by now; any strong emotions should not be obvious anymore.

Reply:
Fair enough.

Reply:
```suggestion
			`	An Elder concludes the meeting. "We have a plan of action on the pirate deals. However, I've heard enough of this 'kidnapping' conspiracy theory. Our meeting is adjourned." Teeneep's fury is palpable, but she remains silent as the three exit.`
```